### DISCUSSION TITLE
i am new in mentorship program please give me some guidence ?

### DISCUSSION THREAD

**Discussion by Aman123lug** [cncf/mentoring#838] [Q&A] [answered]
is there any proposal need to submit contribution?
when this is start? timeline

**Comment by aliok** [comment, marked as answer]
hi @Aman123lug 

Thanks for your interest in GSoC at CNCF.

Best idea right now is checking the GSoC website to get familiar with the program: https://summerofcode.withgoogle.com/

Also, you might want to read the contributor guide: https://google.github.io/gsocguides/student/

Then you can get yourself familiar with the projects listed in the ideas page: https://github.com/cncf/mentoring/blob/main/summerofcode/2023.md

Also, we have a #mentoring channel in CNCF Slack: https://github.com/cncf/mentoring#communication if you would like assistance on a more specific subject.

Timeline for GSoC program is available on the GSoC website.

**Comment by Aman123lug** [comment]
@aliok thank you so much i read all thing from link that given but still have doubts like : cncf gsoc is annouced? , should i start contributing in cncf repository "good first issue"

**Reply by aliok** [reply]
> should i start contributing in cncf repository "good first issue"

Yes, I would encourage you to do that always.

However, please note that there's no good-first-issue in CNCF/mentoring repository. You should check out the projects at the GSoC project ideas [here](https://github.com/cncf/mentoring/blob/main/summerofcode/2023.md) and you can start contributing to those projects listed there.

> cncf gsoc is annouced?

It will be announced by Google in 2 days, if CNCF is an accepted organization or not: https://developers.google.com/open-source/gsoc/timeline#february_22_-_1800_utc
